CH05/PeriodicAndTimeBasedObservables/Program.cs

```diff
@@ -82,55 +74,47 @@ private static void CreatingPeriodicObservableWithInterval()
 
         /// <summary>
         /// Shows how to use the Interval operator to periodically poll a webservice for updates
-        /// NOTE: a similar example for WPF GUI application can be found in the project "CreatingPeriodicUpdatableView" 
+        /// NOTE: a similar example for WPF GUI application can be found in the project "CreatingPeriodicUpdatableView"
         /// </summary>
-        private static void PeriodicallyGetUpdates()
-        {
+        private static void PeriodicallyGetUpdates() {
             Console.WriteLine();
             Demo.DisplayHeader("Using Interval to periodically poll a webservice");
 
             var updatesWebService = new UpdatesWebService();
-            var observable = Observable
+            IObservable<string> observable = Observable
                 .Interval(TimeSpan.FromSeconds(3))
                 .Take(3) // we dont want to run the example forever, so we'll do only 3 updates
                 .SelectMany(_ => updatesWebService.GetUpdatesAsync())
                 .SelectMany(updates => updates);
 
             observable.RunExample("updates");
-
         }
 
-
-        private static void UsingTimerToScheduleWithRelativeTime()
-        {
+        private static void UsingTimerToScheduleWithRelativeTime() {
             Console.WriteLine();
             Demo.DisplayHeader("Schedule single emission with Timer (relative)----");
 
             Console.WriteLine("Scheduling to 5 sec from subscription");
-            var observable = Observable
```

Comment:
please keep the explicit types. it was meant to make it easier for readers to understand the result of the expression

Reply:
yes, received. i will do that at next commit.